Description: Add some example place holders for easing scripted upgrades,
and also for the sake of highlighting the interface change until upstream
documents it.
Author: Adam Gandelman <adamg@canonical.com>

Index: nova-2012.1/etc/nova/api-paste.ini
===================================================================
--- nova-2012.1.orig/etc/nova/api-paste.ini	2012-03-01 19:12:50.000000000 -0800
+++ nova-2012.1/etc/nova/api-paste.ini	2012-03-01 19:16:28.185654630 -0800
@@ -150,8 +150,9 @@
 auth_port = 35357
 auth_protocol = http
 auth_uri = http://127.0.0.1:5000/
-# NOTE: you will have to replace the values below with an actual token
-# or user:password combination.
-# admin_user = %SERVICE_USER%
-# admin_password = %SERVICE_PASSWORD%
 admin_token = %SERVICE_TOKEN%
+# Keystone now authenticates services by service username, password and tenant
+# Update the following with appropriate credentials
+admin_tenant_name = tenant
+admin_user = service_user
+admin_password = service_password
